### DISCUSSION TITLE
Videx 901

### DISCUSSION THREAD

**Discussion by eduardoribeiro** [PricelessToolkit/ESPBell-LITE#2] [Q&A] [answered]
Hi,

I have a Videx 901 Intercom which I would love to make smart, this Intercom has a video feature that I don't plan to integrate since is an analogic video and not digital, apart from this, the Intercom works as most Intercoms, it chimes when someone rings at the building door, open the door and allow 2-way communications and also control the apartment doorbell (it's a different chime though so I believe that the connection is done on a different interface)... 

I already tried to automate the Intercom using Ring Intercom but Ring isn't compatible with it, I managed to get it to work when someone rings at the building door but never was able to make it open the door.

I don't understand much of electric wires on Intercoms so I gave it a look and tried to understand with the technical manual that I found somewhere on the internet, but I'm not sure that the connections mentioned on that PDF are correct. Either way, I looked into that and I wasn't able to connect all the dots (mainly because I lack electronics knowledge regarding this type of connection).

My question is, is the ESPBell_lite or even ESPBell-Max compatible with my Intercom? I have a picture of the Intercom board that I will attach to this discussion and I will attach also the Technical Manual of the Intercom. If so, is there a schematic of the cables that I need to connect to the Intercom to make it work with the ESPBell? 

Photo of the Intercom Board
![IMG_0040](https://github.com/PricelessToolkit/ESPBell-LITE/assets/275310/5334f925-aee7-4fab-bcbe-7ef56551efd6)
An with the connection identified
![1FBE1AF0-CC94-48D4-858D-4D615C0BC285_1_201_a](https://github.com/PricelessToolkit/ESPBell-LITE/assets/275310/9dbb6539-f70a-4227-99b1-7d5d1e987ec0)

Technical Manual
[901mv.pdf](https://github.com/PricelessToolkit/ESPBell-LITE/files/13693329/901mv.pdf)

Picture of electronic schematics from the manual
![Videx901-connections](https://github.com/PricelessToolkit/ESPBell-LITE/assets/275310/98a6142c-92a2-4784-becf-4c6168346cbb)

_Again, I'm not sure that the manual is accurate since I wasn't able to proper determine the model of the Intercom and for the technical drawing in the manual, mine seems to be the 901 (see picture bellow)_
![videx-901](https://github.com/PricelessToolkit/ESPBell-LITE/assets/275310/92a8c90a-1851-4f50-8fa4-038d3b388ea5)

**Comment by PricelessToolkit** [comment]
Hi, I think the problem is connections. Do you have a multimeter? can you test if on terminal 2 voltage is present? normally in 5-wire setup if you connect terminal 2 "door opener" to terminal 6 "negative power" it should open the door. Or vice versa if terminal 2 is not energized, you need to connect it to power. but I can't guarantee anything. PS If Ring can't open ESPbell can't too.

**Reply by eduardoribeiro** [reply]
I don't have a multimeter but I can try to find one and I'll let you know what I find... Ring can open the door maybe because Ring didn't test this device and I made the connection matching Videx 924 that is supported by Ring to this one, but like you said it might be a problem of connections, I might be missing which cables should connect to... Ring contacted me last week asking for more details on my Intercom connections, but I'll try also your suggestions and maybe I'll find what's wrong with it

**Reply by dfrazao** [reply, marked as answer]
I have the exact same intercom and can confirm that connecting terminal 2 to 6 opens the door!

**Reply by eduardoribeiro** [reply]
Are you using the ESPBell or using another solution? @dfrazao can we talk, I would like to achieve a solution to my Intercom to integrate it with Home Assistant.
@PricelessToolkit can I then assume that ESPBell-Lite or Max will work with my Intercom?

**Reply by dfrazao** [reply]
@eduardoribeiro I'm not using anything at this point, but I'm going to buy the ESPBell-Lite. Sure, we can I talk! I was wondering how did you wire your Ring to detect someone ringing the door, that's the main thing I'm not sure how to do.

**Reply by PricelessToolkit** [reply]
@eduardoribeiro  @dfrazao As I understand it correctly, the problem is that the doorbell and buzzer are separate, right? one for the apartment door and the other for the main entrance?

**Reply by PricelessToolkit** [reply]
@dfrazao You have also an optional Buzzer?

**Reply by eduardoribeiro** [reply]
@dfrazao 
I match the schema that Ring provides for Videx 924 to mine... I made these connections using the A cable from Ring:
| Ring | Videx 924 |                     | Videx 901 |                      |
|------|-----------|---------------------|-----------|----------------------|
|      | Port      | Description         | Port      | Description          |
|      | 4         | Electric Call       | 1 ?       | Positive power input |
| A6   | 3         | Call AC             | 6 ?       | Negative power input |
| A2   | 2         | Speech In (Speaker) | 4         | Speech Input         |
| A3   | 1         | Speech Out (Mic)    | 3         | Speech Output        |
| A1   | 3         | Common              | 5 ?       | Speech Ground        |
|      |           |                     | R ?       | Speech Common        |
| A5   | 5         | Lock                | 2         | Lock                 |


The **port 4** on **Videx 924 is not being used by Ring**, so I didn't connect that one but I think it matches **port 1** on **Videx 901**. I have some doubts on ports **A1** which would be the Common (**port 3**) but on Videx 901 **not sure** if that matches **port 5** or **port R**.

I've based on this document to understand the ports for Videx 924 (PDF)[https://interfoni.rs/wp-content/uploads/2019/08/Videx-924-interfonska-slusalica-_-sema-prikljucaka-1.pdf]

@PricelessToolkit yes we do have separate ports for the Bell and the Buzzer, where the Bell is the chime from the outside of the building and the Buzzer is the chime at the apartment door...

**Reply by PricelessToolkit** [reply]
I'll check the ring connection with my intercom, is there a website where I can look for compartability/connection diagram?

**Reply by PricelessToolkit** [reply]
Google search gives me 

- A1 -> COMMON
- A2 -> SPEAKER
- A3 -> MIC
- A4 -> AUDIO COMMON
- A5 -> UNLOCK
- A6 -> DOORBELL

**Reply by eduardoribeiro** [reply]
Not that I'm aware, I got those from the Ring App when setup Ring Intercom selecting Videx 924... anyway, my doubts where regarding which is Doorbell (A6) and Common (A1) on mine alongside with Audio Common (A4)

**Reply by eduardoribeiro** [reply]
@dfrazao you can send me an email at eduardo.ribeiro@outlook.pt or I'll send you an email and we can exchange contacts over email

**Comment by PricelessToolkit** [comment]
i think if connect the "DB optocoupler" to "(6 negativ) and 7" it will detect the doorbell (If the initial voltage between this terminals is 0V)

**Comment by eduardoribeiro** [comment]
@dfrazao I've sent you an email with some of my contacts, feel free to contact me if you will so we can discuss more around each other findings and share some knowledge...

**Comment by eduardoribeiro** [comment]
@PricelessToolkit, by the way I just bought an ESPBell-Lite on your website :-)

**Reply by PricelessToolkit** [reply]
Thanks for your support!!

**Reply by eduardoribeiro** [reply]
I might need support to then connect your ESPBell to my Intercom, meaning I might need help identifying which PINs to connect to each Terminal according to what we discussed, I've ordered the power supply already so once I received I'll give it a try and I'll let you know

**Reply by PricelessToolkit** [reply]
OK, no problem!

**Comment by dfrazao** [comment]
@PricelessToolkit Can you tell by looking at the page from the manual that @eduardoribeiro posted, if it would be possible to get power directly from the intercom, using a buck converter by connecting it to the 12v terminal for positive and terminal 6 for negative?

**Reply by PricelessToolkit** [reply]
@dfrazao In theory, yes BUT, I don’t recommend doing this, the intercom main transformer may burn out, and the entire building system will fail.

**Reply by dfrazao** [reply]
Ok, I also have the HLK-PM01 power supply, how would you recommend wiring it?

**Reply by PricelessToolkit** [reply]
1. For HLK-PM01  220V side as short as possible "for safety reasons" don't put 220v into your intercom. 2. use any 5V phone charger.

**Reply by eduardoribeiro** [reply]
@dfrazao are you trying to assemble the ESPBell-Lite with your Intercom? I was supposed to received my ESPBell today but I wasn't home so most probably only tomorrow. And I'm still waiting on the Power Supply. But I'll probably need some insights on connections if you can then provide some pics of the connections and what you did would be awesome

**Reply by dfrazao** [reply]
@eduardoribeiro I got it working like this:
![image](https://github.com/PricelessToolkit/ESPBell-LITE/assets/26813660/d10c042a-036c-4b38-8e95-d512f99e0486)
Each lock on the ESPBell has two wires, I used a wire from each for the lock, otherwise, it will keep the lock button "stuck".
From my testing with a multimeter, I think that the apartment buzzer could also be integrated into this, But I haven't tested it using the ESPBell.

And for power, I had an issue @PricelessToolkit I couldn't get it working using the GND and VCC on the bottom of the board, it just didn't powered on, I've been using the 3v3 and GND pads on the back instead. Do you have any ideas of what I could try to do to fix it?

**Comment by dfrazao** [comment]
I had an issue powering the board, @PricelessToolkit I couldn't get it working using the GND and VCC on the bottom of the board, it just didn't power on, I've been using the 3v3 and GND pads on the back instead. Do you have any ideas of what I could try to do to fix it?

**Reply by PricelessToolkit** [reply]
When I test the ESPBell-Lite after production, I use a connector with a cable and 5V. Therefore, it should work if the regulator is not burned out or damaged during transportation. Was the box intact? Could you please take a photo of this part?
![pcbpart](https://github.com/PricelessToolkit/ESPBell-LITE/assets/81021972/84a78c78-7365-4f18-8b99-4f5887d3aa15)

**Reply by dfrazao** [reply]
That's what I tried too, I checked using a multimeter on the cable, and I was getting around 5V..  Yes, the box was intact. Here's the photo:
![2024-01-05-11-53-36-421](https://github.com/PricelessToolkit/ESPBell-LITE/assets/26813660/b57fe9f7-4bab-4959-9e89-6f4042bd9e6b)

**Reply by PricelessToolkit** [reply]
can you test voltage across 2 big capacitors when it powered with 5V

**Reply by PricelessToolkit** [reply]
I hope you have noticed that the colors red and black on the cable do not mean the usual; here, red is GND, and black is VCC.

**Reply by dfrazao** [reply]
I got it working! I think it was just the cable making bad contact :/
